### ISSUE TITLE
Unknown replication status/last sync

### ISSUE THREAD

**Issue by ngaruko** [medic/medic#5414]
_Brief description of bug encountered_.

**Steps to reproduce**:
 - _Log in as restricted user_
 - _Open the options/breadcrumb navigation _
 - _look at the `replication status`_

**What should happen**:
 - _Should show last sync time and replication status_
![image](https://user-images.githubusercontent.com/6979995/52921170-25380b00-3379-11e9-9611-4361237c4c6e.png)

**What actually happens**:
 - _Shows `Unkown`_
![image](https://user-images.githubusercontent.com/6979995/52921183-5b758a80-3379-11e9-9678-5e445c36857f.png)
and this in console:
```Replicate to server successful with unknown seconds since the previous successful replication.```

Then I went to FF and cleared cache, cookies and offline website data...
Now, it shows proper last sync time, but status is `Currently syncing...` forever
![image](https://user-images.githubusercontent.com/6979995/52921215-9f688f80-3379-11e9-83ea-fb33367881ed.png)

(*Also seen on mobile phone*)
**Environment**:

- Instance: _(gamma.dev.medicmobile.org)_
- Browser: _(Firefox, Chrome)_
- Client platform: _( MacOS)_
- App: _(webapp, Mobile app)_
- Version: _(3.3.0)_

**Comment by dianabarsan** [comment]
Merged into `3.3.x`. `3.4.x` replication mechanism is different and likely not affected.

**Comment by newtewt** [comment]
Tested on gamma.dev with 3.3.x branch installed. This is now showing the updated sync time instead of unknown and console shows the timing as well. 
```
Replicate to server successful with 308.331 seconds since the previous successful replication.
inbox.js:3 Replicate to server successful with 2.958 seconds since the previous successful replication.
inbox.js:3 Replicate from server successful with 10.343 seconds since the previous successful replication.
```

![image](https://user-images.githubusercontent.com/10531066/52971860-02145800-3387-11e9-89ed-46c8ed5057e6.png)
